testing/e2e/suites/utils.go

```diff
@@ -41,7 +41,7 @@ func waitForAvailable(ctx context.Context, c client.Client, deployment appsv1.De
 func upsert(ctx context.Context, c client.Client, obj client.Object) error {
 	copy := obj.DeepCopyObject().(client.Object)
 	lgr := logger.FromContext(ctx).With("object", copy.GetName(), "namespace", copy.GetNamespace())
-	lgr.Info("upserting object")
+	lgr.Info(fmt.Sprintf("upserting object: %s", obj.GetName()))
```

Comment:
no need to add the name here. it's already attached to the logger on the line above this.

Reply:
in general attaching to the fields like on 43 here is better if we want more information anyways since it guarantees the context exists in the entire relevant code block. we don't want to have to duplicate relevant information in all logs in a block